### DISCUSSION TITLE
依依 K 歌 | /posts/yi-yi-sing/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#605] [Announcements]
https://yuanfan.rbind.io/posts/yi-yi-sing/

**Comment by residualsun1** [comment]
本00后表示也没有看剧的习惯，因为可能一看就停不下来，干脆就不看了。

另，我个人觉得《Love Story》和《认真的雪》还是算比较新的流行歌诶。

> 又到吃饭时间了，真烦，为什么人不能不吃饭呀。

同问。虽然但是，我还是会觉得经常做饭比较麻烦，别人经常给自己做自己还过意不去……

**Reply by earfanfan** [reply]
啊，00后，你好：

  本90后并不是没有看剧的习惯，只是看不进去那些粗制滥造的工业流水线产品而已。你这干脆不看的理由听起来很厉害啊。

  另，这两首歌只是出现在你生命里的的时间距离现在很新，可是出现在我记忆中的时间已是许多年前了。

  关于觉得做饭麻烦这点，由于本社畜工作日都得带饭，所以总在琢磨怎样能更快更方便操作，渐渐就不觉得麻烦了。而且自己做饭久了，好像就会变得吃不惯外面餐馆的菜，我也不知道这是为撒（ps我还发愁朋友来武汉玩的话该带去哪找好吃的……）。关于别人给自己做饭自己会过意不去这件事我没遇见过，如果不是你总在强迫别人为你奉献付出的那种极端情况的话，也许你可以认为自己值得得到别人的付出，就不感到过意不去是不是也可以。

**Reply by residualsun1** [reply]
> 你这干脆不看的理由听起来很厉害啊。

阿，其实是因为没啥抵抗力，所以只能粗暴地隔绝。我现在把微信朋友圈关闭，将B站和豆瓣都转移到不经常带在身边的平板，大大降低了使用这些软件的频率。当然，我偶尔也会瞄一两眼。可能这个做法看起来会比较极端，不过慢慢地，我发现对自己的现实生活更加投入了——这是我想要的效果。

> 关于觉得做饭麻烦这点，由于本社畜工作日都得带饭，所以总在琢磨怎样能更快更方便操作，渐渐就不觉得麻烦了。

原来你还会自备饭菜上班，我觉得这样挺好的。在我刻板印象里，上班族都是结束工作后拖着疲惫的身躯到餐馆吃饭……我也不大喜欢餐馆的饭菜，喜欢家常菜，一般能吃三四碗饭（惊人的饭量吧）。关于为啥不喜欢吃外面的菜，我家里人最常见、最简单的说法就是家里和外面用的油都完全不一样，外面的菜不香。

> 如果不是你总在强迫别人为你奉献付出的那种极端情况的话，也许你可以认为自己值得得到别人的付出，就不感到过意不去是不是也可以。

嗯，我支持^^。

**Reply by CyrusYip** [reply]
> 虽然但是，我还是会觉得经常做饭比较麻烦，别人经常给自己做自己还过意不去……

@residualsun1 可能是你不爱做饭又必须做饭？我以前对做饭不感兴趣的时候也有这样的心态。如果你是这么想的，那可能觉得别人给你做饭也是苦差事。对某些人（比如家庭主妇）来说，给别人做饭（服务他人）就会感到幸福。

**Reply by residualsun1** [reply]
@CyrusYip 对，我心态上会有“不爱做饭又必须做饭”的感觉。我在民宿里待着的时候，如果客人要和我们吃饭，为了保证质量，都是别人做。时间长了，我感觉自己不做点啥心里就有点不大过得去。可能这和我觉得自己没能给那儿的小伙伴帮上忙有关，不过现在更明确自己的定位，调整了心态，感觉比之前要好了一些。

> 对某些人（比如家庭主妇）来说，给别人做饭（服务他人）就会感到幸福。

我觉得会有诶，做饭对他们来说可能是一种愉快的事情，特别是看到别人把自己的饭菜吃光。我在民宿的几个小伙伴中饭量是最大的，不过大家也不嫌弃，都很乐意把我“喂”饱，越看我吃饭越觉得饭菜香……
